README.md

```diff
@@ -24,3 +24,54 @@ end
 3. Commit your changes (`git commit -am 'Add some feature'`)
 4. Push to the branch (`git push origin my-new-feature`)
 5. Create new Pull Request
+
+## Testing
+
+### General: Which Ruby
+
+Check that the Ruby version you are running is one that is supported.
+You can see what versions RSpec is tested against by looking at the
+[.travis.yml](https://github.com/rspec/rspec-support/blob/v3.2.1/.travis.yml#L16-26)
+(noting [allowed failures](https://github.com/rspec/rspec-support/blob/master/.travis.yml#L34-L36))
+and the [appveyor.yml](https://github.com/rspec/rspec-support/blob/v3.2.1/appveyor.yml#L32-L34).
+
+### Running tests
+
+`bundle exec rake` locally, and check the CI results in your pull request.
+
```

Comment:
I didn't see that a place for this yet exists on http://rspec.info/contributing/.  If something like this text is ok, I would make a PR to add it to the page and link to it from here.